### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by twolfson**
We are moving our code that relies on `email-split` from JavaScript to Python. As a result, we need to port the code between languages. In this PR:
- Added tests
- Added library
- Added docs

A preview of the README can be viewed here:

https://github.com/underdogio/python-email-split/tree/dev/initial.release

/cc @brettlangdon

### CODE REVIEW

File: email_split/__init__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        # domain is content after `@` in email
+        self.local = local
+        self.domain = domain
+
+    @classmethod
+    def split_email(cls, email):
+        """Break up an email and initialize a new class"""
+        # Break up our email at its `@`
+        # https://docs.python.org/2/library/re.html#re.split
+        email_parts = re.split(r'@', email, maxsplit=1)
```
Comment (brettlangdon) [inline email_split/__init__.py]: could always use partition here as well:

``` python
local, _, domain = email.partition('@')
```

Comment (brettlangdon) [inline email_split/__init__.py]: Or, just not using `re`,

``` python
# DEV: `split` does not accept keyword arguments for `maxsplit`
email_parts = email.split('@', 1)
```

**Comment by brettlangdon** [comment]
small suggestion, but lgtm

File: email_split/__init__.py (hunk 1 above)
Comment (twolfson) [inline email_split/__init__.py]: TIL about `partition`. Will update to use that https://docs.python.org/2/library/stdtypes.html#str.partition

**Comment by twolfson** [comment]
Updated PR to use `partition`, will land once Travis CI approves

**Comment by twolfson** [comment]
Ugh, I was fucking with the history due to `master` not having a `+x` `release.sh`

**Comment by twolfson** [comment]
This has been released as `1.0.0` (with some hiccups in `foundry` -- e.g. missing `.foundryrc` and `+x release.sh`)

**Pull request closed without merging**